src/install/install

```diff
@@ -0,0 +1,20 @@
+echo "Installing sourcerer app.."
+
+if [ -f /usr/local/bin/sourcerer ] ; then
+  read -e -p "Previous sourcerer app installation is detected. Reinstall it? [Y/n] " yesno < /dev/tty
+  if [ "$yesno" ] && [ "$yesno" = "n" ] ; then
+    echo "Exit.."
```

Comment:
Exiting wihout .. or with ..? Otherwise I confirm that I didn't miss this comment and I think it is reasonable.